docs/developer-guide/frontend-development.md

```
@@ -77,6 +77,31 @@ The UI uses OIDC against the in-cluster Dex. After opening `http://localhost:809
 
 All passwords are the literal string `password`. Cluster RBAC bindings live in `test/fixtures/e2e/dex/dex-rbac.yaml` (inlined from `docs/developer-guide/manifests/`).
 
+## Testing against the remote Zitadel
+
+Production authenticates against ZITADEL, not Dex. To point the dev UI at it:
+
+```bash
```

markdownlint-cli2 (0.23.2)

[warning] Code block style
Expected: indented; Actual: fenced

(MD046, code-block-style)